sea.js

```diff
@@ -168,24 +171,31 @@
 
     if(SEA.window){
       api.crypto = window.crypto || window.msCrypto;
+      if(!api.crypto) {
+        api.crypto = require('isomorphic-webcrypto');
+      }
       api.subtle = (api.crypto||o).subtle || (api.crypto||o).webkitSubtle;
       api.TextEncoder = window.TextEncoder;
-      api.TextDecoder = window.TextDecoder;
+      api.TextDecoder = window.TextDecoder;      
       api.random = (len) => Buffer.from(api.crypto.getRandomValues(new Uint8Array(Buffer.alloc(len))))
     }
+    if(!api.TextDecoder)
+    {
+      const { TextEncoder, TextDecoder } = require('text-encoding')
+      api.TextDecoder = TextDecoder
+      api.TextEncoder = TextEncoder
+    }
```

Comment:
Let's add semicolons here, as seems to be convention elsewhere mostly (my bad initially).

Reply:
done